template/Dockerfile/images/nginx.jinja2

```diff
@@ -14,11 +14,14 @@
         nginx
 {%- endmacro %}
 
-
-{% macro debian(role='') -%}
+{% macro debian(role='',osCodename='buster') -%}
     # Install nginx
+    && printf "deb http://nginx.org/packages/debian/ {{ osCodename }}  nginx\n deb-src http://nginx.org/packages/debian/ {{ osCodename }} nginx" \
+    >> /etc/apt/sources.list \
+    && curl -L https://nginx.org/keys/nginx_signing.key | apt-key add - \
+    && apt-get update \
```

Comment:
You don't need `apt-get update` our `apt-install` command already includes that. See https://github.com/webdevops/Docker-Image-Baselayout/blob/master/root/usr/local/bin/apt-install

Reply:
Okay thanks, didn't see that!